src/css/theme/_toggle.scss

```diff
@@ -0,0 +1,17 @@
+.ais-toggle {
+  &--item {
+    @extend .ais-current-refined-values--item;
+
+    &__active {
```

Comment:
The react-is theme does bold the count pill when the item is selected, example: https://community.algolia.com/react-instantsearch/storybook/?selectedKind=HierarchicalMenu&selectedStory=with%20default%20selected%20item&full=0&down=1&left=1&panelRight=1&downPanel=storybooks%2Fstorybook-addon-knobs